search/search_quality/assessment_tool/main_model.hpp

```diff
@@ -27,20 +27,30 @@ class MainModel : public Model
 
   // Model overrides:
   void Open(std::string const & path) override;
+  void Save() override;
+  void SaveAs(std::string const & path) override;
```

Comment:
No, this is an override of the method from the interface, and string was included in the model.hpp